client/my-sites/importer/newsletter/content-upload/conversion-summary.tsx

```diff
@@ -0,0 +1,291 @@
+					__(
+						'You had some image file types that we do not support. You will need to convert any %s files to a different format and upload them.'
+					),
+					fileEntries.map( ( [ ext ] ) => '<strong>' + ext + '</strong>' ).join( ', ' )
+			  ) ) + ' ';
+
+	return (
+		<div className="conversion-summary__unsupported-files">
+			<p>
+				{ createInterpolateElement(
+					message + __( '<supportLink>Learn more about the supported file types.</supportLink>' ),
+					{
+						supportLink: (
+							<InlineSupportLink
+								supportLink={ localizeUrl(
+									'https://wordpress.com/support/accepted-filetypes/#images'
+								) }
+								showIcon={ false }
+								supportPostId={ 2037 }
+							/>
+						),
+						strong: <strong />,
+					}
+				) }
+			</p>
+		</div>
+	);
+};
+
+const weHaveBlocksForTheseMap: {
+	[ key: string ]: {
+		nodeName: string;
+		blockName: string;
+		icon: JSX.Element;
+		supportLink: string;
+		supportPostId: number;
+	};
+} = {
+	'native-video-embed': {
+		nodeName: 'Native video embed',
+		blockName: 'Video block',
+		icon: video,
+		supportLink: localizeUrl(
+			'https://wordpress.com/support/wordpress-editor/blocks/video-block/'
+		),
+		supportPostId: 149045,
+	},
+	'native-audio-embed': {
+		nodeName: 'Native audio embed',
+		blockName: 'Audio block',
+		icon: audio,
+		supportLink: localizeUrl(
+			'https://wordpress.com/support/wordpress-editor/blocks/audio-block/'
+		),
+		supportPostId: 148670,
+	},
+	'digest-post-embed': {
+		nodeName: 'Embedded post',
+		blockName: 'Embed block',
+		icon: post,
+		supportLink: localizeUrl(
+			'https://wordpress.com/support/wordpress-editor/blocks/embed-block/'
+		),
+		supportPostId: 150644,
+	},
+	'poll-embed': {
+		nodeName: 'Poll embed',
+		blockName: 'Poll block',
+		icon: help,
+		supportLink: localizeUrl( 'https://wordpress.com/support/wordpress-editor/blocks/poll-block/' ),
+		supportPostId: 170183,
+	},
+	'latex-rendered': {
+		nodeName: 'LaTeX content',
+		blockName: 'LaTeX block',
+		icon: code,
+		supportLink: localizeUrl( 'https://wordpress.com/support/latex/' ),
+		supportPostId: 97,
+	},
+	'image-gallery-embed': {
+		nodeName: 'Image gallery',
+		blockName: 'Image Gallery block',
+		icon: gallery,
+		supportLink: localizeUrl(
+			'https://wordpress.com/support/wordpress-editor/blocks/gallery-block/'
+		),
+		supportPostId: 148667,
+	},
+};
+
+const PostErrorsMessage = ( { postErrors }: { postErrors: PostErrorsType } ) => {
+	const nodeFailedErrors = postErrors.convert_node_failed || {};
+	const hasNodeFailedErrors = Object.keys( nodeFailedErrors ).length > 0;
+
+	const gutenbergFailedErrors = postErrors.convert_html_to_gutenberg_failed;
+
+	if ( ! hasNodeFailedErrors && ! gutenbergFailedErrors ) {
+		return null;
+	}
+
+	return (
+		<div className="conversion-summary__post-errors">
+			{ gutenbergFailedErrors && (
+				<p>
+					{ __(
+						'We found some posts that had no content in them. There is nothing to do here, just thought you should know.'
```

Comment:
I wonder if the phrasing may be a bit confusing to ESL readers, and translation may be more difficult. Can we remove the second part maybe?


```suggestion
						'We found some posts that had no content in them and that could not be imported.'
```